sycl/test-e2e/lit.cfg.py

```diff
@@ -0,0 +1,435 @@
+
+acc_run_substitute = "true"
+acc_check_substitute = ""
+if 'acc' in config.target_devices.split(','):
+    found_at_least_one_device = True
+    lit_config.note("Tests accelerator device")
+    acc_run_substitute = " env ONEAPI_DEVICE_SELECTOR='*:acc' "
+    acc_check_substitute = "| FileCheck %s"
+    config.available_features.add('accelerator')
+else:
+    lit_config.warning("Accelerator device not used")
+config.substitutions.append( ('%ACC_RUN_PLACEHOLDER',  acc_run_substitute) )
+config.substitutions.append( ('%ACC_CHECK_PLACEHOLDER',  acc_check_substitute) )
+
+if config.sycl_be == 'ext_oneapi_cuda' or (config.sycl_be == 'ext_oneapi_hip' and config.hip_platform == 'NVIDIA'):
+    config.substitutions.append( ('%sycl_triple',  "nvptx64-nvidia-cuda" ) )
+elif config.sycl_be == 'ext_oneapi_hip' and config.hip_platform == 'AMD':
+    config.substitutions.append( ('%sycl_triple',  "amdgcn-amd-amdhsa" ) )
+else:
+    config.substitutions.append( ('%sycl_triple',  "spir64" ) )
+
+if find_executable('sycl-ls'):
+    config.available_features.add('sycl-ls')
+
+# TODO properly set XPTIFW include and runtime dirs
+xptifw_lib_dir = os.path.join(config.dpcpp_root_dir, 'lib')
+xptifw_dispatcher = ""
+if platform.system() == "Linux":
+    xptifw_dispatcher = os.path.join(xptifw_lib_dir, 'libxptifw.so')
+elif platform.system() == "Windows":
+    xptifw_dispatcher = os.path.join(config.dpcpp_root_dir, 'bin', 'xptifw.dll')
+xptifw_includes = os.path.join(config.dpcpp_root_dir, 'include')
+if os.path.exists(xptifw_lib_dir) and os.path.exists(os.path.join(xptifw_includes, 'xpti', 'xpti_trace_framework.h')):
+    config.available_features.add('xptifw')
+    config.substitutions.append(('%xptifw_dispatcher', xptifw_dispatcher))
+    if cl_options:
+        config.substitutions.append(('%xptifw_lib', " {}/xptifw.lib /I{} ".format(xptifw_lib_dir, xptifw_includes)))
+    else:
+        config.substitutions.append(('%xptifw_lib', " -L{} -lxptifw -I{} ".format(xptifw_lib_dir, xptifw_includes)))
+
+
+llvm_tools = ["llvm-spirv", "llvm-link"]
+for llvm_tool in llvm_tools:
+  llvm_tool_path = find_executable(llvm_tool)
+  if llvm_tool_path:
+    lit_config.note("Found " + llvm_tool)
+    config.available_features.add(llvm_tool)
+    config.substitutions.append( ('%' + llvm_tool.replace('-', '_'),
+                                  os.path.realpath(llvm_tool_path)) )
+  else:
+    lit_config.warning("Can't find " + llvm_tool)
+
+if find_executable('cmc'):
+    config.available_features.add('cm-compiler')
+
+# Device AOT compilation tools aren't part of the SYCL project,
+# so they need to be pre-installed on the machine
+aot_tools = ["ocloc", "opencl-aot"]
+
+for aot_tool in aot_tools:
+    if find_executable(aot_tool) is not None:
+        lit_config.note("Found pre-installed AOT device compiler " + aot_tool)
+        config.available_features.add(aot_tool)
+    else:
+        lit_config.warning("Couldn't find pre-installed AOT device compiler " + aot_tool)
+
```

Comment:
Just as a note, this will need to be updated with the changes from https://github.com/intel/llvm-test-suite/pull/1535 when kernel fusion tests are moved.